pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -320,34 +324,61 @@ func (l *LibvirtDomainManager) FinalizeVirtualMachineMigration(vmi *v1.VirtualMa
 	return l.finalizeMigrationTarget(vmi)
 }
 
-// HotplugHostDevices attach host-devices to running domain
-// Currently only SRIOV host-devices are supported
+// HotplugHostDevices attach host-devices to running domain, currently only SRIOV host-devices are supported.
+// This is operation runs on the background each at a time.
```

Comment:
Can you please clarify this comment?
*in the background

Reply:
Done